servers/rendering/renderer_scene_cull.h

```diff
@@ -1054,6 +1055,8 @@ class RendererSceneCull : public RenderingMethod {
 	virtual void instance_set_visible(RID p_instance, bool p_visible);
 	virtual void instance_geometry_set_transparency(RID p_instance, float p_transparency);
 
+	virtual void instance_teleport(RID instance);
```

Comment:
```suggestion
	virtual void instance_teleport(RID p_instance);
```
Please prefix arguments with `p_`